blackbox/docs/admin/ssl.rst

```diff
@@ -396,11 +396,11 @@ Output::
     Certificate was added to keystore
 
 
-Import CA Into Truststore
+Import CA into truststore
 .........................
 
 If we are using our own CA, we should also import the certificate to the
-Truststore, such that it is available for clients which want to verify
+truststore, such that it is available for clients which want to verify
```

Comment:
```suggestion
Truststore, such that it is available for clients which want to verify
```

Reply:
done

Reply:
This change wouldn't relate to the PR. Let's fix this in a separate one.

Reply:
in this case, your PR changes the case of "Truststore" to "truststore". my suggested change reverts that